doc/getting_started.md

```diff
@@ -0,0 +1,36 @@
+# Getting started
+Now that you have successfully installed Azure CLI 2.0 and added the Azure DevOps extension, you are all set to get started!

```

Comment:
`Azure CLI 2.0` I am not sure if putting version is a good idea. What if azure CLI 3.0 is released tomorrow?

Reply:
Azure CLI 1.0. refers to classic and 2.0 refers to the new CLI and there is a confusion when refered to Azure CLI on whether it is classic or not. 2.0 version is used for clarity. We can follow with the version as when it changes which I assume won't be frequent. Further, if it does, we'll further have to include version referece to provide further compatibility clarity.